.clang-tidy

```diff
@@ -6,6 +6,7 @@ Checks: >
   performance-*,
   portability-*,
   readability-*,
+  -modernize-use-trailing-return-type
```

Comment:
This will remove that annoying change of trailing return on main (and future methods/functions we'll have)